package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-brightspace",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
Can you make this `0.10.0`? A patch release will get pushed out to everyone immediately and start failing their builds, which we'd like to avoid. A minor release with a `0.x` major release "acts" like a major release (semver is weird).